templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -1,13 +1,13 @@
 {{if .ctxData.IsSigned}}
 <div class="item action ui dropdown jump pointing top right select-reaction" data-action-url="{{.ActionURL}}">
-	<a class="add-reaction">
+	<a role="button" class="add-reaction">
 		{{svg "octicon-smiley"}}
 	</a>
 	<div class="menu">
-		<div class="header">{{.ctxData.locale.Tr "repo.pick_reaction"}}</div>
+		<div class="header reactions-header">{{.ctxData.locale.Tr "repo.pick_reaction"}}</div>
 		<div class="divider"></div>
 		{{range $value := AllowedReactions}}
-			<a class="item reaction" data-tooltip-content="{{$value}}" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
+			<a class="item reaction" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
```

Comment:
The old `data-tooltip-content` is also for `aria-label`, if it is removed, a new `aria-label` attribute should be set.

Reply:
Added it back as `title` and `aria-label`.